CHANGES

```diff
@@ -23,6 +23,9 @@ Deprecated
 ----------
 
 * #11247: Deprecate the legacy ``intersphinx_mapping`` format
+* #11165: The ``_t`` suffix for templating (``sphinx.util.copy_asset()``,
+  ``sphinx.util.copy_asset_file()`` and built-in templates) is now deprecated.
```

Comment:
Ok, yep: I'll update the `CHANGES` entry (and this pull request title) to say "pending deprecation" instead.

Sorta-related: reading back through some of the previous commentary, it looks like I misread one of your previous messages:

>> I was wondering though: since it seems to be reasonably simple to support both _t and .jinja for a deprecation period, why not support both indefinitely and never deprecate them?
>
> That seems like a good approach to me :+1: 

Somehow I didn't read/understand the `never deprecate them` part of that.

To me, the goal here is to make the purpose and content of the template files less ambiguous.  Currently `_t` indicates that they're templates, but doesn't say much about what kind of templating that is.  `.jinja` indicates both template, and a templating language that's in use.

(so, correcting myself: supporting both sounds good, but I would like to suggest deprecating `_t` at some point)

Reply:
If we supported both suffixes `_t` and `.jinja` equally: would there be any situations where we'd recommend using `_t` instead of `.jinja`?